include/nbl/builtin/glsl/virtual_geometry/descriptors.glsl

```diff
@@ -21,13 +21,13 @@
 #define _NBL_VG_UINT_BUFFERS_COUNT 0
 #endif
```

Comment:
the default of `_NBL_VG_FLOAT_BUFFERS_COUNT ` is a tad extreme

Reply:
8?

Reply:
position, color, uv, normal = 4